lib/spack/spack/build_environment.py

```diff
@@ -569,22 +569,10 @@ def _static_to_shared_library(arch, compiler, static_lib, shared_lib=None,
     return compiler(*compiler_args, output=compiler_output)
 
 
-def get_rpath_deps(pkg):
-    """Return immediate or transitive RPATHs depending on the package."""
-    if pkg.transitive_rpaths:
-        return [d for d in pkg.spec.traverse(root=False, deptype=('link'))]
-    else:
-        return pkg.spec.dependencies(deptype='link')
-
-
 def get_rpaths(pkg):
     """Get a list of all the rpaths for a package."""
-    rpaths = [pkg.prefix.lib, pkg.prefix.lib64]
```

Comment:
@scheibelp these changes as still OK IMO as that's what CMakePackage relies on https://github.com/spack/spack/blob/develop/lib/spack/spack/build_systems/cmake.py#L167